src/install.rs

```diff
@@ -360,21 +374,6 @@ async fn install_actions<'a>(
     Ok(0)
 }
 
-fn file_manager(config: &Config, fm: &str, pkgs: &[&str]) -> Result<()> {
```

Comment:
Why not just make this return the view instead of ripping out the whole function?

Reply:
`view ` type is `TempDir`, need 
```

Reply:
`view` type is `TempDir`. to return view, we need

```rust
install.rs:
extern crate tempdir;
use tempdir::TempDir;

Cargo.toml:
[dependencies]
tempdir = "0.3.7"

Reply:
Is it okay now?

Reply:
Yep that looks good to me, thanks.